cloudinit/util.py

```diff
@@ -3275,3 +3276,23 @@ def decorator(*args, **kwargs):
         return decorator
 
     return wrapper
+
+
+def read_hotplug_enabled_file() -> dict:
+    content: dict = {"scopes": []}
+    try:
+        content = json.loads(
+            load_text_file(settings.HOTPLUG_ENABLED_FILE, quiet=False)
+        )
+    except FileNotFoundError:
+        pass
```

Comment:
We should debug log here. The `load_text_file` is going to log `Reading from /var/lib/cloud/hotplug.enabled` regardless, so it appears the file was read when it actually wasn't.

Reply:
Good point, log added.